deploy/helm/edot-collector/kube-stack/values.yaml

```diff
@@ -841,75 +518,418 @@ collectors:
+        attributes/dataset:
+          actions:
+            - key: event.dataset
+              from_attribute: data_stream.dataset
+              action: upsert
+        batch:
+          send_batch_size: 1000
+          timeout: 1s
+          send_batch_max_size: 1500
+        # [Elastic Trace Processor](https://github.com/elastic/opentelemetry-collector-components/tree/main/processor/elastictraceprocessor)
+        elastictrace: {} # The processor enriches traces with elastic specific requirements.
+        # [LSM Interval Processor](https://github.com/elastic/opentelemetry-collector-components/tree/main/processor/lsmintervalprocessor)
+        lsminterval:
+          intervals:
+            - duration: 1m
+              statements:
+                - set(resource.attributes["metricset.interval"], "1m")
+                - set(attributes["data_stream.dataset"], Concat([attributes["metricset.name"], "1m"], "."))
+                - set(attributes["processor.event"], "metric")
+            - duration: 10m
+              statements:
+                - set(resource.attributes["metricset.interval"], "10m")
+                - set(attributes["data_stream.dataset"], Concat([attributes["metricset.name"], "10m"], "."))
+                - set(attributes["processor.event"], "metric")
+            - duration: 60m
+              statements:
+                - set(resource.attributes["metricset.interval"], "60m")
+                - set(attributes["data_stream.dataset"], Concat([attributes["metricset.name"], "60m"], "."))
+                - set(attributes["processor.event"], "metric")
+      exporters:
+        debug: {}
+          # [Elasticsearch exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/elasticsearchexporter/README.md)
+        elasticsearch/otel:
+          endpoints: # List of Elasticsearch endpoints.
+            - ${env:ELASTIC_ENDPOINT}
+          api_key: ${env:ELASTIC_API_KEY} # API key for Elasticsearch authentication.
+          logs_dynamic_index:
+            enabled: true
+          # Enable in order to skip the SSL certificate Check
+          # tls:
+          #   insecure_skip_verify: true
+          mapping:
+            mode: otel
+        elasticsearch/ecs:
+          endpoints:
+            - ${env:ELASTIC_ENDPOINT}
+          api_key: ${env:ELASTIC_API_KEY}
+          mapping:
+            mode: ecs
+      service:
+        pipelines:
+          metrics:
+            receivers: [otlp]
+            exporters: [routing]
+          metrics/infra/ecs:
+            receivers: [routing]
+            processors:
+              - elasticinframetrics
+              - attributes/dataset
+              - batch
+            exporters: [debug, elasticsearch/ecs]
+          metrics/infra/otel:
+            receivers: [routing]
+            processors: [batch]
+            exporters: [debug, elasticsearch/otel]
```

Comment:
Why don't we send metrics to `metrics/otel`, so that the `signaltometrics` connector is invoked for these?

Reply:
Good point, fixed in https://github.com/elastic/elastic-agent/pull/6444/commits/8d859d62048a7349fe394cdd1aa3f37bace0f874

Should we route the ecs infra metrics to the `signaltometrics` connector too?

Reply:
Good question. Since we already route the `otel` metrics through it, it feels like that would duplicate the counting. That's why I'd lean towards not sending `ecs` metrics through the `signaltometrics` connector.